.stylish-haskell.yaml

```diff
@@ -11,7 +11,7 @@ steps:
       empty_list_align: inherit
       list_align: new_line
       list_padding: 4
-      long_list_align: new_line_multiline
+      # long_list_align: new_line_multiline
```

Comment:
Hi @paolino @Unisay 

Apologies for the intrusion, but for the GHC/node bump, is it necessary to comment out this line?

It seems that the `long_list_align: new_line_multiline` option is still supported by the most recent `stylish-haskell`. (See https://github.com/haskell/stylish-haskell/blob/4cde3c606b318d7112fdbddb60853e0c59d8433a/lib/Language/Haskell/Stylish/Step/Imports.hs#L124)

**My concerns:**

1. Commenting out this line seems to change the formatting and style of imports for all Haskell files in `cardano-wallet`, even those that are completely unrelated to the node/GHC bump.

2. This causes conflicts with several other in-flight PRs that add/remove imports. If we merge this PR as is, then we'd need to perform additional work to rebase those other PRs and fix them up. (I'd prefer to not do that, if possible!)

**Gentle request:**

1. Assuming the node/GHC bump does not require us to change the import style, then can we avoid changing it in this PR? 🙏🏻

2. If this change was made purely from a desire to change the style, then could we discuss the proposed style change in a dev meeting after we're all back?

(The import style changes seem to be located in a separate commit, so perhaps we can simply avoid merging that commit.)

Forgive me if I've missed some of the context behind this decision, or if there's a very good reason for making the change that I'm unaware of.

All the best!

Jonathan

Reply:
Ah, after a bit of digging, I found @paolino's comment on the `stylish-haskell` repo here:

https://github.com/haskell/stylish-haskell/issues/462

Reply:
Yeah, it's awful, I hope we can move to fourmolu ASAP

Reply:
We manage to roll back the change and still have stylish-haskell 0.11.0.3 in scope

Reply:
> We manage to roll back the change and still have stylish-haskell 0.11.0.3 in scope

Wow, that's great news! 🍾

Thanks! ❤️